_docs_integrate/terminate-relationships.md

```diff
@@ -0,0 +1,56 @@
+
+## Reactivate a terminated Relationship
+
+At any time, you or your peer can request the reactivation with the use case [Request Relationship reactivation]({% link _docs_use-cases/use-case-transport-request-relationship-reactivation.md %}).
+
+If your peer requests the reactivation, you will receive a `transport.relationshipReactivationRequested` [event]({% link _docs_integrate/connector-events.md %}), which the peer also receives if they use a Connector.
+If you then wish to return the Relationship to an active state, you accept the reactivation with the use case [Accept Relationship reactivation]({% link _docs_use-cases/use-case-transport-accept-relationship-reactivation.md %}).
+If you want to keep the Relationship terminated, you reject the reactivation with the use case [Reject Relationship reactivation]({% link _docs_use-cases/use-case-transport-reject-relationship-reactivation.md %}).
+
+If you have requested the reactivation and changed your mind, you revoke it with the use case [Revoke Relationship reactivation]({% link _docs_use-cases/use-case-transport-revoke-relationship-reactivation.md %}).
+
+Each use case has the relationshipId as input. Only accepting the reactivation changes the `status` of the Relationship (from `"Terminated"` to `"Active"`), requesting/rejecting/revoking the reactivation don't change it. However, they still are recorded in the [`auditLog` of the Relationship]({% link _docs_integrate/data-model-overview.md %}#auditlogentry). In each case (accept/reject/revoke), you receive a `transport.relationshipReactivationCompleted` [event]({% link _docs_integrate/connector-events.md %}) and the peer as well if they use a Connector.
+
+## Decompose a Relationship
+
+Decomposing a Relationship deletes from your Connector
+
+- the Relationship
+- the peer's RelationshipTemplates
+- the Relationship's [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) if it was single use (its `maxNumberOfAllocations` is 1) or owned by the peer - for a new Relationship you will have to exchange a Template again
+- shared [Attributes]({% link _docs_integrate/data-model-overview.md %}#attributes), [Notifications]({% link _docs_integrate/data-model-overview.md %}#notification), [Requests]({% link _docs_integrate/data-model-overview.md %}#request) (in each case both sent and received)
+- sent and received [Messages]({% link _docs_integrate/data-model-overview.md %}#message) with one exception: If you have sent a message to multiple recipients, the message is not deleted but the peer's address is replaced with a pseudonym. The pseudonym is the same for every peer.
+- more obscurely, [Tokens]({% link _docs_integrate/data-model-overview.md %}#token) and [AttributeListeners]({% link _docs_integrate/data-model-overview.md %}#localattributelistener) (if you haven't had an opportunity to learn what those mean, it doesn't matter for you).
```

Comment:
> (if you haven't had an opportunity to learn what those mean, it doesn't matter for you)

this sounds like a punch in the face for the reader.

Reply:
also - why are Tokens and AttributeListeners **obscure**?

Reply:
Tokens are not mentioned in the main explanatory pages of Integrate, only in the data model and use cases - and I didn't see that AttributeListeners were in the "Working with Requests" section, but it's still less visible than in the "Manage Attributes" section

I'm linking to the data model by convention, which is overly stuffed for a first contact with the topic, so I wanted people to merrily go along if they don't know them

Reply:
but don't do that this aggressive.

Reply:
IMO the reader skips uninteresting content anyways